src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -236,6 +239,11 @@ public Command sysIdDynamic(SysIdRoutine.Direction direction) {
         return m_sysIdRoutineToApply.dynamic(direction);
     }
 
+    // sets all motors' neutral modes to coast (false) or brake (true)
+    public void setBrake(boolean brake) {
+            this.configNeutralMode(brake ? NeutralModeValue.Brake : NeutralModeValue.Coast);
```

Comment:
nit, this is 2 indents instead of the one
```suggestion
      this.configNeutralMode(brake ? NeutralModeValue.Brake : NeutralModeValue.Coast);
```